TESTING.md

```diff
@@ -0,0 +1,76 @@
+# Testing Tracetest
+
+On Tracetest, we work in two ways to test the entire system and guarantee that everything is working fine. One is through automatic tests, where we run unit, integration and end-to-end tests on Tracetest CLI, Web UI and API server.
+
+Another source of tests is the manual tests that we execute on each release, simulating users, and checking if everything is ok on these tests.
+
+## Manual Tests
+
+On our manual tests, we aim to do some [sanity checks](https://en.wikipedia.org/wiki/Sanity_check) to see if the main features are working as expected. Usually, we run it on each Tracetest release.
+
+### Testing Tracetest setup
+
+This is a simple test to check if Tracetest is working correctly given it was provisioned with one Tracing Data Store.
+
+The steps that we should follow are:
+
+- [ ] Open WebUI and go to `/settings` page. The provisioned Data Store should be selected.
+- [ ] Run `tracetest datastore export -d {provisioned_datastore}` and check if the data was exported correctly.
+- [ ] Create a test on WebUI that calls a demo API (like [Pokeshop](https://docs.tracetest.io/live-examples/pokeshop/overview) or [Open Telemetry Store](https://docs.tracetest.io/live-examples/opentelemetry-store/overview)). This test should fetch traces correctly and run without errors.
+
+### Checklist on version release
+
+This is the entire checklist on what we should do to assert that Tracetest is working fine on each version release. On each version release, we can copy the contents of this checklist and open a Github Discussion to start each test.
+
+- [ ] Check if our release pipeline on [Release Tracetest](https://github.com/kubeshop/tracetest/actions/workflows/release-version.yml) workflow on Github Actions worked correctly.
+- [ ] Test CLI update on MacOS via homebrew
```

Comment:
These tests are good candidates to be automated in the future with docker, creating test scripts that tries to install Tracetest locally in a SO docker image.